### ISSUE TITLE
Challenge Sorted Union has a bug and does not validate while I'm having the good results

### ISSUE THREAD

**Issue by liorchamla** [FreeCodeCamp/FreeCodeCamp#11444]
Challenge [Sorted Union](https://www.freecodecamp.com/challenges/sorted-union#?solution=function%20unite(arr1%2C%20arr2%2C%20arr3)%20%7B%0A%20%20return%20Array.from(arguments).reduce(function(all%2C%20current)%7B%0A%20%20%20%20return%20all.concat(current)%3B%0A%20%20%7D%2C%20%5B%5D).filter(function(element%2C%20index%2C%20array)%7B%0A%20%20%20%20return%20array.indexOf(element)%20%3D%3D%3D%20index%3B%0A%20%20%7D)%3B%0A%7D%0A%0Aunite(%5B1%2C%202%2C%203%5D%2C%20%5B5%2C%202%2C%201%2C%204%5D%2C%20%5B2%2C%201%5D%2C%20%5B6%2C%207%2C%208%5D)%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
function unite(arr1, arr2, arr3) {
  return Array.from(arguments).reduce(function(all, current){
    return all.concat(current);
  }, []).filter(function(element, index, array){
    return array.indexOf(element) === index;
  });
}

unite([1, 2, 3], [5, 2, 1, 4], [2, 1], [6, 7, 8]);

```

**Comment by systimotic** [comment]
Hey @liorchamla! Your code is indeed correct. The issue is that it appears you renamed your function. The tests are looking for a function named `uniteUnique`, but your function is named `unite`. Changing back the function name makes the challenge pass. If you need coding help, please click the "help" button on the challenge.
I'm closing this issue. If you believe I misunderstood the problem, feel free to add more information and reopen the issue.

**Comment by liorchamla** [comment]
No you were right bro. The problem was the function name. But the weird thing is that I did not modify the function name. Maybe the exercise has changed between the time I did it in February and now. And so my code was good at this time but not anymore.

I found the same problem on the next challenge (old code that used to work but now doesn't anymore),  I just did a reset, and rewrite the same code, then it worked. I figured out the reset has changed something, maybe the function's name.

Anyway, this is fixed !

gg wp ;-)
